src/impl/load.c

```diff
@@ -20,6 +20,29 @@ char *get_xt_gcg_string(const char *identifier, ErrorStack *error_stack) {
 
   // Check if this is a Cross-tables URL
   if (strstr(identifier, "cross-tables.com") != NULL) {
+      // Check if it's a direct GCG URL first
+      if (strstr(identifier, "cross-tables.com/annotated/selfgcg") != NULL) {
```

Comment:
This one's my bad, I should have clarified that I meant a URL from any source, not necessarily cross-tables, so we should probably move this to it's own "get_url_gcg_string" function or something.